src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SpecialCommunity.java

```diff
@@ -0,0 +1,14 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.communitypage;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+public class SpecialCommunity extends WikiBasePageObject {
+
+  public static String COMMUNITY_PAGE_URL = "Special:Community";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make COMMUNITY_PAGE_URL a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "public static COMMUNITY_PAGE_URL" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1444)